addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -209,7 +216,26 @@ private synchronized void connect() {
         discoTime = config.discoveryTimeout > 0 ? config.discoveryTimeout : DISCO_TIME;
         heartBeatInterval = config.heartBeatInterval > 0 ? config.heartBeatInterval : HEARTBEAT_INTERVAL;
 
-        discovery = new HarmonyHubDiscovery(discoTime);
+        String host = config.host;
+
+        // earlier versions required a name and used network discovery to find the hub and retrieve the host,
```

Comment:
do you really think we should keep that code for all time? Wouldn't it be better on the long run to get rid off that?
The whole code could become so much cleaner if the handler wouldn't also deal with the discovery service...